packages/paste-website/src/pages/core/changing-theme.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Changing the Theme
+description: When using Paste, migrating to a different theme is a simple two step process.
+slug: /core/changing-theme
+---
+
+import {graphql} from 'gatsby';
+import {SidebarCategoryRoutes} from '../../constants';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/core/changing-theme/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  description={props.pageContext.frontmatter.description}
+  categoryRoute={SidebarCategoryRoutes.CORE}
+/>
+
+---
+
+ <contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Update the theme provider
+
+The `<Theme.Provider>` is a wrapper component that provides a theme object to any descendant. It is usually towards the root of your app.
+
+```jsx
+import {Theme} from '@twilio-paste/theme';
+
+<Theme.Provider theme="default">
+  <App />
+</Theme.Provider>;
+```
+
+To change the theme, simply change the value of the `theme` prop on the `Theme.Provider`. If you are using Paste component correctly and only usign the tokens to style your app, the `<Theme.Provider>` handles (almost) everything for you.
+
+## Load the font
+
+**For the default theme:** the best way to load the fonts is to include the following snippet in the `<head />` of your web page.
+
+```html
+<link rel="preconnect" href="https://assets.twilio.com" crossorigin />
+<link rel="stylesheet" href="https://assets.twilio.com/public_assets/paste-fonts/main-1.2.0/fonts.css" />
+```
+
+Alternatively, Paste will automatically load the default theme's font via JavaScript. Note: this will result in slower download times than including the fonts in the `<head />`.
```

Comment:
Should the "Note" be called out a bit more formally?  Bold / new paragraph / Callout?

Reply:
I'll make it bold. I took most of the content from this page: https://paste.twilio.design/getting-started/manual-installation#default-theme